csunplugged/utils/TranslatableModel.py

```diff
@@ -29,7 +29,7 @@ def get_queryset(self):
 class TranslatableModel(models.Model):
     """Abstract base class for models needing to store list of available languages."""
 
-    languages = ArrayField(models.CharField(max_length=5), default=[])
+    languages = ArrayField(models.CharField(max_length=10), default=[])
```

Comment:
Reason for this increase?

Reply:
Not all language codes are xx or xx_yy format (eg. Chinese simplified: zh_Hans). None supported by django are longer than 10 (see https://github.com/django/django/tree/master/django/conf/locale).

Catching it before it becomes a problem later.